packages/block-library/src/gallery/style.scss

```diff
@@ -41,7 +42,9 @@
 		img {
 			display: block;
 			height: auto;
-			max-width: 100%;
+			// Ensure max-width is not overridden on the img when the parent gallery has
+			// wide or full alignment.
+			max-width: 100% !important;
```

Comment:
This specifically was necessary for the frontend on Twenty Nineteen, to override:
```
@media only screen and (min-width: 768px)
.entry .entry-content .wp-block-image:not(.alignwide):not(.alignfull) > img {
    max-width: calc(8 * (100vw / 12) - 28px);
```
As the `alignfull`/`alignwide` class is present on the container but not included on the `img` children.